tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/field_persist_test.dart

```diff
@@ -388,18 +383,11 @@ void main() {
       StatefulAnalyzer(config, models, onErrorsCollector(collector))
           .validateAll();
 
-      expect(collector.errors.length, greaterThan(1));
-
-      var error1 = collector.errors[0];
-      var error2 = collector.errors[1];
+      expect(collector.errors.length, 1);
```

Comment:
chore: this test is reduntant remove it. Please check all other tests. We don't have any mutually exclusivity anymore so there is nothing to validate.